Eevee.Sleep.Bot/Handlers/EventHandlers/GuildMemberJoinedEventHandler.cs

```diff
@@ -0,0 +1,19 @@
+using Discord.WebSocket;
```

Comment:
Shouldn't need this, just add the role record of a user when their data didn't exist before.

Reply:
same as this
https://github.com/RaenonX-PokemonSleep/eevee-sleep-discord-bot/pull/25#discussion_r1606034397